src/metatensor/models/cli/conf/architecture/experimental.alchemical_model.yaml

```diff
@@ -16,6 +16,9 @@ training:
   batch_size: 8
   num_epochs: 100
   learning_rate: 0.001
+  patience: 50
```

Comment:
I would name it `early_stopping_patience`, because at first sight I was confused as to what this was (since there was already a scheduler patience). Also, I feel like 100 is a better default value for most cases, but I don't have much experience fitting either model.

If we really wanted to go overkill, we could set it to e.g. 1/10th of `num_epochs`, but this would be for a different PR